docs/release-notes/8.3.asciidoc

```diff
@@ -14,6 +87,78 @@
+
+NOTE: Use curl or another HTTP tool to run {elastic-sec} API commands.
+
+Here is a curl command that you can run to find enabled rules:
+
+[source,console]
+--------------------------------------------------
+GET /api/detection_engine/rules/_find?per_page=10000&filter=alert.attributes.enabled:true
+--------------------------------------------------
+
+If you already upgraded from 8.2 to 8.3.0, detection rules affected by the bug will have stopped running with an error that is similar to the following example:
+
+[source,text]
+----
+<rule-type>:<UUID>: execution failed - security_exception: [security_exception] Reason: missing authentication credentials for REST request [/_security/user/_has_privileges], caused by: ""
+----
+
+You will need to disable, then re-enable affected rules to restore them to a functional state. These actions generate a new API key using the credentials of the user performing these actions. The rules' states are also reset.
+
+NOTE: To restore failed custom and prebuilt rules, you need privileges to <<enable-detections-ui, manage rules>>.
+
+*Restore custom rules*
+
+Follow these steps:
+
+. Go to the *Rules* page (*Detect -> Rules*) and click *Custom rules*.
+. Sort the rules table by the *Enabled* column to show enabled rules.
+. Select the affected rules, then click *Bulk actions -> Disable*.
++
+NOTE: Affected rules will have a `Failed` execution status and the aforementioned error message in the rule's details.
+
+. Select the affected rules, then click *Bulk actions -> Tags -> Add Tags*. Add a descriptive tag, for example `rules_to_fix`.
+. Re-select the same rules, then click *Bulk actions -> Enable*.
+. Add a descriptive tag, for example `rules_to_fix`. Adding a tag will generate new API keys and solve the issue.
+. Follow these steps if you don't want to wait until the next scheduled rule execution (optional):
+.. From the rules table, filter for the tag you just applied to your custom rules.
+.. Click *Select all x rules* above the rules table.
+.. Click *Bulk actions -> Disable*.
+.. Filter the rules table for the same tag, re-select the same rules, and then click *Bulk actions -> Enable*.
+
+Once the rule runs, the *Last Response* value for the rule will change to `Pending`, and then to `Active` or `OK`.
+
+*Restore prebuilt rules*
+
+Follow these steps:
+
+. Use the <<rules-api-find, Find rules API>> to capture a list of enabled rules before you upgrade. Here is a curl command that you can run to find enabled rules:
++
+[source,console]
+--------------------------------------------------
+GET /api/detection_engine/rules/_find?per_page=10000&filter=alert.attributes.enabled:true
```

Comment:
Should this command match the one in [line 64](https://github.com/elastic/security-docs/pull/2181/files#diff-d5e679f0881b8afa92631d68e73db22e9f91b39186a7ad283ac469a068d30d23R64), since they're both looking for prebuilt rules? I think this command would find any enabled rules, both prebuilt and custom, but please confirm with dev.

Reply:
This should match what's on line 64 and you are correct that this command find both types of enabled rules (prebuilt and custom). Good catch, thanks!

```suggestion
GET api/detection_engine/rules/_find?page=1&per_page=1000&sort_field=name&sort_order=asc&filter=alert.attributes.enabled:true and alert.attributes.params.immutable: true
```